pdcupdater/handlers/modules.py

```diff
@@ -0,0 +1,274 @@
+import os
+import logging
+import errno
+import re
+from subprocess import check_call, STDOUT
+
+import beanbag
+import pdcupdater.handlers
+import pdcupdater.services
+from pdcupdater.utils import TmpDir, PushPopD
+
+import modulemd
+
+
+log = logging.getLogger(__name__)
+
+md_ns = {
```

Comment:
@ralphbean, good catch! This is from a very early version of my handler that pulled info from repository metadata. I'll remove it.